src/localizers/globalize.js

```diff
@@ -37,13 +37,28 @@ export let formats = {
 export default function(globalize) {
   let locale = culture => culture ? globalize(culture) : globalize;
 
+  // return the first day of the week from the locale data. Defaults to 'world'
+  // territory if no territory is derivable from CLDR.
+  // Failing to use CLDR supplemental (not loaded?), revert to the original
+  // method of getting first day of week.
   function firstOfWeek(culture) {
-    let date = new Date();
-    //cldr-data doesn't seem to be zero based
-    let localeDay = Math.max(
-      parseInt(locale(culture).formatDate(date, { raw: 'e' }), 10) - 1, 0)
-
-    return Math.abs(date.getDay() - localeDay)
+    try {
+        const days = ['sun', 'mon', 'tue', 'wed', 'thu', 'fri', 'sat'];
```

Comment:
sorry for my lack of experience here, but will these be english for every culture?